registries/_format/base64url.md

```diff
@@ -0,0 +1,33 @@
+---
+owner: baywet
+issue:
+description: A url-safe binary array as defined in RFC4648
+base_type: string
+layout: default
```

Comment:
```suggestion
layout: default
oas_version: < 3.1
remarks: "When using OpenAPI 3.1 or above it's recommended not to use this format and instead use [`contentEncoding` with a value of `base64url`](https://json-schema.org/draft/2020-12/json-schema-validation.html#name-contentencoding)."
```

Reply:
for this one I'm not sure the markdown in a yaml field will be parsed properly by the templating engine. Would you mind if we keep it to plain text?

Reply:
Not at all

Reply:
```suggestion
layout: default
oas_version: < 3.1
remarks: "When using OpenAPI 3.1 or above it's recommended not to use this format and instead use contentEncoding with a value of base64url."
```

Reply:
actually it looks like MD works, let me revert to what you originally suggested